Makefile

```diff
@@ -310,13 +310,15 @@ benchmark:
 
 lint:
 	golangci-lint run --out-format=tab --issues-exit-code=0
-	find . -name '*.go' -type f -not -path "./vendor*" -not -path "*.git*" -not -name '*.pb.go' | xargs gofmt -d -s
```

Comment:
Please revert. This must fail if and only if the code is not `gofmt`'d (line 312 never fails).

Reply:
this is not needed, `golangci-lint` handle this faster (because it reuses parsed files). So better way is to fail in 312.

Reply:
Otherwise there is no much point of using a linter.

Reply:
OK makes sense